src/scripts/dungeons/Dungeon.ts

```diff
@@ -9604,7 +9604,6 @@ dungeonList['Kalos Power Plant'] = new Dungeon('Kalos Power Plant',
                 requirement: new MultiRequirement([
                     new ClearDungeonRequirement(5, GameConstants.getDungeonIndex('Kalos Power Plant')),
                     new GymBadgeRequirement(BadgeEnums.Elite_KalosChampion),
-                    new QuestLineStepCompletedRequirement('A Beautiful World', 11),
```

Comment:
A Beautiful World 11 requires the player to beat some Team Flare folks in here. Removing this makes old save able to progress the quest from defeating Volcanion instead. The quest should be turned into DefeatDungeonBoss before removing the requirement.

Reply:
I have commented a TODO in the associated questline step to turn it into a dungeon boss quest for when the quest refactor is merged, because the story questline isn't supposed to be relevant to the unlock requirement of Volcanion, and shouldn't be shown as a hint. For clarity's sake I will add another note in this file too. 👍